public/blog/2023-11-dask-survival-analysis/2023-11-dask-survival-analysis.md

```diff
@@ -0,0 +1,48 @@
+---
+author: "Jaka Kokošar"
+date: "2023-11-08"
+draft: false
+title: "From Data Portals to Portals of Doom: Avoiding the Doom with Dask"
+url: "dask-survival-analysis"
+thumbImage: "orange-not-responding.png"
+frontPageImage: "orange-not-responding.png"
+blog: ["dask", "survival analysis"]
+shortExcerpt: "Sidestep the limitations of web-based visual anlaytics tools and and steer your survival analysis with confidence and ease. Dive into the entire TCGA cohort within a single Orange workflow."
+longExcerpt: "Sidestep the limitations of web-based visual anlaytics tools and and steer your survival analysis with confidence and ease. Dive into the entire TCGA cohort within a single Orange workflow."
+---
+
+Since its inception, The Cancer Genome Atlas (TCGA) project has been a treasure trove of data for biomedical researchers. The data TCGA provides is essential for researchers who want to analyze potential biomarkers. But there's a hitch. Not every researcher is a data whisperer.
+
+Luckily, there are several web-based tools that researchers can add to their toolkits. These tools make it easy to access and analyze survival data from TCGA. Notable examples are [Kaplan-Meier Plotter](https://kmplot.com/analysis/) and [GEPIA](http://gepia.cancer-pku.cn/), among [others](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2020.00068/full).
+
+<WindowScreenshot src="gepia-example.png" />
+
+In their very essence, these tools allow you to enter a gene name and output the results of the analysis on pre-curated TCGA datasets. For example, in the image above, I have entered the gene name TP53 and selected the cancer type lower-grade glioma (TCGA-LGG). The output is the Kaplan-Meier plot for the high- and low-risk patient groups. I can also see the p-value of the log-rank test, which tells me how significant the difference in survival between the groups is. For a detailed description of functionalities, I suggest you read this review paper by [Zheng et al.](https://www.frontiersin.org/journals/oncology/articles/10.3389/fonc.2020.00068/full).
+
+
+Perfect, this is just what I need. So where is the problem? Well, as it turns out, you can't just blindly trust the underlying source of survival data of those tools. Curated datasets between tools can vary (more in [this paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8726696/) by Idogawa et al.). And this is no small thing. How can you trust the analysis if you can trust the data source? Adding fuel to the fire here, I will let the reader guess how many of those tools are open-sourced.
```

Comment:
Perfect, this is just what I need. So where is the problem? -> Look good, where is the catch?  (I tried making it less verbose, maybe you'll have a better idea.

Reply:
you can't -> we can't

Reply:
" How can you trust the analysis if you can trust the data source?" -> mogoče lapsus? Je sploh to potrebno še enkrat omeniti, glede na to, da je že v prvem odstavku?